pkg/daemon/config.go

```diff
@@ -19,6 +19,13 @@ import (
 	ptpv1 "github.com/k8snetworkplumbingwg/ptp-operator/api/v1"
 )
 
+// predefined config section names
+const (
+	GlobalSectionName  = "[global]"
+	NmeaSectionName    = "[nmea]"
+	UnicastSectionName = "[unicast_master_table]"
+)
```

Comment:
no considering other sections, for an example in syncE ?
[global]
logging_level               7
use_syslog                  0
verbose                     1
message_tag [synce4l.0.config]
smc_socket_path             /run/synce4l_socket
[<synce1>]
module_name                 ice
dnu_prio  0XFF
network_option              2
extended_tlv                1
recover_time                60
clock_id  5799633565433967748
[ens7f0]
tx_heartbeat_msec           1000
rx_heartbeat_msec           10
allowed_qls                 0xE
allowed_ext_qls             0x40
[{GNSS}]
board_label                 GNSS-1PPS
input_QL                    0x1
input_ext_QL                0x20

Reply:
`[global]
logging_level               7
use_syslog                  0
verbose                     1
message_tag [synce4l.0.config]
smc_socket_path             /run/synce4l_socket
[<synce1>]
module_name                 ice
dnu_prio  0XFF
network_option              2
extended_tlv                1
recover_time                60
clock_id  5799633565433967748
[ens7f0]
tx_heartbeat_msec           1000
rx_heartbeat_msec           10
allowed_qls                 0xE
allowed_ext_qls             0x40
[{GNSS}]
board_label                 GNSS-1PPS
input_QL                    0x1
input_ext_QL                0x20
`